### ISSUE TITLE
Build Lugaru without CMake

### ISSUE THREAD

**Issue by dressupgeekout** [dressupgeekout/lugarudc#11]
Getting CMake to handle a KOS-based project is probably as simple as `CC = kos-cc, CXX = kos-c++`, etc. I don't mind CMake in general, but I don't want to fight any Dreamcast battles with CMake if I don't have to.

In general, Lugaru's CMakeLists is pretty restrained and doesn't do anything too stupid. I should be able to write a plain Makefile instead, which in my opinion gives me a LOT more direct control over the compiler and linker command-lines.

TL;DR Luguaru utilizes CMake to maximize portability, but CMake definitely doesn't know or care what the Dreamcast is, so I shouldn't bother with it.

**Comment by dressupgeekout** [comment]
Done 2f372bb6ac01bcadc1bc30b2f909c67d7d80fac4
